modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/common/RasDetailsQueryParams.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package dev.galasa.framework.api.ras.internal.common;
+
+public class RasDetailsQueryParams {
+    
+    private String detail;
```

Comment:
I'm wondering if this should be a `Set<String>` that we populate in the constructor using the query param's string value rather than a string

Probably fine for now since we only have "methods"